src/shared/icons/styles.scss

```diff
@@ -0,0 +1,12 @@
+.icon { //need classname
+	width: 1em;
+	height: 1em;
+}
+
+	.preview {
+		position: absolute;
+		top: 50%;
+		left: 50%;
+		font-size: 128px;
+		transform: translate(-50%, -50%);
```

Comment:
ahhh, I get what you're doing here, but this definitely belongs in the styles for episode preview - this icon could theoretically be pulled out and used anywhere - in which case it doesn't need to know about preview styles, y'know? Just make this `s.icon` in `/EpisodePreview/styles.scss` and pass that down I reckon.

Reply:
Ok fair enough. I've copied the styles into s.icon for EpisodePreview. Had to specify a height for the div wrapper as it otherwise was taller than it was wide, for no good reason 🤷‍♂